frontstage_api/controllers/django_controller.py

```diff
@@ -28,12 +28,10 @@ def sign_in(username, password):
     response = request_handler('POST', url, headers=headers, auth=app.config['DJANGO_BASIC_AUTH'], data=data)
 
     if response.status_code == 401:
-        logger.debug('Authentication error in oauth2 service')
         oauth2_error = json.loads(response.text)
-        raise ApiError(url, response.status_code, oauth2_error)
+        raise ApiError(url, response.status_code, oauth2_error, description='Authentication error in oauth2 service')
```

Comment:
Should this also be as above and oauth2_error=oauth2_error?

Reply:
This is a slightly different case, this is getting passed back as the `data` parameter which actually gets sent back through to frontstage rather than just being logged.
Might be a good place to be explicit and write `data=oauth2error` though